MongoDB.FSharp.nuspec

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>MongoDB.FSharp</id>
-    <version>0.1.0</version>
-    <title>MongoDB.FSharp</title>
-    <authors>Tim Kellogg</authors>
-    <owners>Tim Kellogg</owners>
-    <licenseUrl>https://github.com/tkellogg/MongoDB.FSharp/blob/master/License.txt</licenseUrl>
-    <projectUrl>https://github.com/tkellogg/MongoDB.FSharp</projectUrl>
-    <!--<iconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>-->
+    <id>MongoDB.Driver.FSharp</id>
+    <version>0.2.0</version>
+    <title>MongoDB.Driver.FSharp</title>
+    <authors>Sebastian Faltoni, Tim Kellogg</authors>
+    <owners>Sebastian Faltoni</owners>
```

Comment:
Sorry i was going to remove the modification on owners tags my intent was the authors tags